frontend/packages/client/src/App.sass

```diff
@@ -1,23 +1,17 @@
-@import url('https://fonts.googleapis.com/css?family=Arimo:regular,bold,italic&subset=latin,latin-ext')
-@import url('https://fonts.googleapis.com/css?family=Roboto+Mono:regular,bold,italic&subset=latin,latin-ext')
+@import url('https://fonts.googleapis.com/css?family=DM+Sans:regular,bold,italic&subset=latin,latin-ext')
 @charset "utf-8"
 
 // Set brand colors
-$green: #0E5B20
-$light-green: #00EF8B
-$red: #B20D0D
-$error-red: #B20D0D
+$success: #3EAE4F
+$danger: #F54339
 $yellow: #FBD84D
 $orange: #F4AF4A
-$light-gray: #DCDCDC
-$lighter-dark-grey: #757575
+$light-grey: #DCDCDC
+$grey: #636363
 
 // Update Bulma's global variables
-$family-primary: "Arimo"
-$family-secondary: "Arimo"
-$family-code: "Roboto Mono"
-$family-monospace: "Roboto Mono"
-$title-family: $family-secondary
+$family-sans-serif: "DM Sans"
+$family-monospace: "DM Sans"
```

Comment:
I think this class also needs to be updated:
```
.image-caption-draft-js
	.public-DraftStyleDefault-block.public-DraftStyleDefault-ltr
		margin: 0px !important
		font-family: 'Arimo'
		font-style: normal
		font-weight: 400
		font-size: 12px
		line-height: 140%
		color: #757575
`````
maybe remove the `font-family: 'Arimo'`